packages/vscode/src/webview-loader.ts

```diff
@@ -81,6 +100,68 @@ export default class WebviewLoader implements vscode.Disposable {
     });
   }
 
+  private getCurrentWorkspacePath() {
+    const currentWorkspaceUri = vscode.workspace.workspaceFolders?.[0].uri;
+    if (!currentWorkspaceUri) {
+      throw new WorkspacePathUndefinedError("Cannot find current workspace path");
+    }
+    return normalizeFsPath(currentWorkspaceUri.fsPath);
+  }
+
+  private async setGitPath() {
+    this.gitPath = (await findGit()).path
+  }
+  
+  private async fetchBranches() {
+    try {
+      return await getBranches(this.gitPath, this.getCurrentWorkspacePath());
+    } catch (e) {
+      console.debug(e);
+    }
+  }
+
+  private async fetchCurrentBranch() {
+    let branchName;
+    try {
+      branchName = await getCurrentBranchName(this.gitPath, this.getCurrentWorkspacePath());
+    } catch (error) {
+      console.error(error);
+    }
+
+    if (!branchName) {
+      const branchList = (await this.fetchBranches())?.branchList;
+      branchName = getDefaultBranchName(branchList || []);
+    }
+    return branchName;
```

Comment:
```suggestion
    try {
      const currentBranchName = await getCurrentBranchName(this.gitPath, this.getCurrentWorkspacePath());
      if (!currentBranchName) {
        const branchList = (await this.fetchBranches())?.branchList;
        return getDefaultBranchName(branchList || []);
      }
      return currentBranchName;
    } catch (error) {
      console.error(error);
    }
```
형식 차이긴 한데 const만 사용하려고 하면 이렇게 쓸수도 있겠네요!